brokerapi/brokers/broker_base/broker_base.go

```diff
@@ -35,7 +36,7 @@ type BrokerBase struct {
 
 // Bind creates a service account with access to the provisioned resource with
 // the given instance.
-func (b *BrokerBase) Bind(instanceID, bindingID string, details brokerapi.BindDetails) (models.ServiceBindingCredentials, error) {
+func (b *BrokerBase) Bind(ctx context.Context, instanceID, bindingID string, details brokerapi.BindDetails) (models.ServiceBindingCredentials, error) {
 	return b.AccountManager.CreateCredentials(instanceID, bindingID, details, models.ServiceInstanceDetails{})
```

Comment:
The ctx should probably be passed down through the AccountManager.CreateCredentials() call as well. It looks like down the call stack in createServiceAccount() there's still a use of background ctx on line 171.